mne/beamformer/tests/test_ap.py

```diff
@@ -0,0 +1,112 @@
+# Authors: Yuval Realpe <yuval.realpe@gmail.com>
+#
+# License: BSD-3-Clause
+
+import os.path as op
+
+from random import choice
```

Comment:
can you use numpy for this? so it used the number generator approach?

Reply:
it also make your test deterministic

Reply:
changed 'from random import choise' to 'from numpy.random import choice'.
reagarding the deterministic nature of the test - I don't fully understand what you mean by that, It chooses different values on a uniform distribution, but I might be missing somthing.

Reply:
The idea is to do

```py
rng = np.random.default_rng(seed)
rng.choice(...)
```
By setting the seed you will get the same choice every time the test runs so you never get a failure just because of the randomness.

Reply:
Thanks. decided to import _get_data() from test_rap.py which doesn't use randomness in test and other wise is (almost) the same.